presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -579,7 +581,11 @@ public Void visitWindow(WindowNode node, Integer indent)
 
             for (Map.Entry<Symbol, WindowNode.Function> entry : node.getWindowFunctions().entrySet()) {
                 FunctionCall call = entry.getValue().getFunctionCall();
-                print(indent + 2, "%s := %s(%s)", entry.getKey(), call.getName(), Joiner.on(", ").join(call.getArguments()));
+                String frameInfo = "";
```

Comment:
If you make `formatFrame` take a `WindowFrame` (see comments below), you can do this:

```java
 String frameInfo = call.getWindow()
    .flatMap(Window::getFrame)
    .map(PlanPrinter::formatFrame)
    .orElse("");
```